tests/Feature/ChurchPresenterTest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Tests\Feature;
+
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Foundation\Testing\WithFaker;
+use Tests\TestCase;
+
+use App\Church;
+
+class ChurchPresenterTest extends TestCase
+{
+  use RefreshDatabase;
+
+  protected $churches;
+  protected $church_1;
+  protected $church_2;
+
+  public function setUp(): void
+  {
+    parent::setUp();
+    $this->church_1 = factory(Church::class)->create();
+    $this->church_2 = factory(Church::class)->create();
+    $this->churches = collect([$this->church_1, $this->church_2]);
```

Comment:
I'm curious what does `collect` do?

Reply:
collect defines a collection, which is what a where query returns